front-end/src/screens/Discussions/index.tsx

```diff
@@ -14,6 +15,25 @@ const Discussions = ({ className } : {className?: string}) => {
 		history.push('/post/create');
 	};
 
+	useState(() => {
+		var infoVisible = JSON.parse(localStorage?.getItem('infoVisible')!);
```

Comment:
instead of setting discussionsInfoVisible onchainInfoVisible in a object then stringifying/parsing it for setting/getting into localstorage you can just do 

localStorage.setItem('discussionsInfo', 'visible')
localStorage.getItem('discussionsInfo') === 'visible'

much cleaner that way

Reply:
Ah, thanks. That's how I had it first, but wasn't sure if an info object to have all that information in makes sense for later on. Changing it back for now :)

Reply:
My bad, I told you the object way. Either way isn't super costly.